ruby/vampires.rb

```diff
@@ -1,82 +1,115 @@
 # Create a program that asks a series of questions,
 	# Assign the input to a variable
+puts "How many employees would you like to process today?"
+employee_count = gets.to_i
 
-print "What is your name?"
-employee_name = gets.chomp
-case employee_name
-when "Drake Cula"
-	vamp_handle = true
-when "Tu Fang"
-	vamp_handle = true
-else 
-	vamp_handle = false
-end
+index = 0
 
-print "How old are you?" 
-age = gets.to_i
+while index < employee_count
 
-print "What year were you born?"
-birth_year = gets.to_i
-current_year = 2016
+	puts "What is your name?"
+	employee_name = gets.chomp
+	case employee_name
+	when "Drake Cula"
+		vamp_handle = true
+	when "Tu Fang"
+		vamp_handle = true
+	else 
+		vamp_handle = false
+	end
 
-if current_year - birth_year === age
-	real_age = true
-else
-	real_age = false
-end
+	puts "How old are you?" 
+	age = gets.to_i
 
-print "Our company cafeteria serves garlic bread. Should we order some for you?"
-garlic = gets.chomp
-case garlic
-when "yes"
-	likes_garlic = true
-when "no"
-	likes_garlic = false
-end
-		
+	puts "What year were you born?"
+	birth_year = gets.to_i
+	current_year = 2016

```

Comment:
with this setup you will have to change this file every year for it to work.

Check out the Ruby `Time` class. You should be able to handle a dynamic year check with something like `Time.new.year`